app/javascript/src/Navigation/components/ContextSelector.jsx

```diff
@@ -11,13 +11,14 @@ import { createReactWrapper } from 'utilities/createReactWrapper'
 
 import 'Navigation/styles/ContextSelector.scss'
 
-import type { Api, Service, Menu } from 'Types'
+import type { Api, Menu } from 'Types'
 
 type Props = {
   apis: Api[],
   currentApi: Api,
   activeMenu: Menu,
-  audienceLink: string
+  audienceLink: string,
+  apiap: boolean
```

Comment:
Better make this optional not to affect existing code?
```suggestion
  apiap?: boolean
```